lua/lspconfig/server_configurations/sourcery.lua

```diff
@@ -0,0 +1,49 @@
+local util = require 'lspconfig/util'
+
+local root_files = {
+  'pyproject.toml',
+  'setup.py',
+  'setup.cfg',
+  'requirements.txt',
+  'Pipfile',
+  'pyrightconfig.json',
+}
+
+return {
+  default_config = {
+    cmd = { 'sourcey', 'lsp' },
+    filetypes = { 'python' },
+    init_options = {
+      editor_version = 'vim',
+      extension_version = 'vim.lsp',
+      token = '<YOUR_TOKEN>',
```

Comment:
I think we should a remove the token section, and instead require that people use the `auth.yaml` file in the `~/.config/sourcery/auth.yaml` file. Sourcery even gives the user the command: `sourcery login` to do this.

Reply:
Can it not work with a local `auth.yaml`? otherwise, where's the path for Windows? I can't find that info on their site, could you please share the link?

Reply:
I mean, I guess it can. But from what I see, I can't understand where the token is used, besides the variable assignment 
```lua
token = "<YOUR_TOKEN>"
```
Unless I am wrong, and it is used inside the mechanics of lspconfig?

Reply:
> Can it not work with a local `auth.yaml`? otherwise, where's the path for Windows? I can't find that info on their site, could you please share the link?

What is considered a "local" `auth.yaml` file? 
Also, output from `sourcery login --help`:
<img width="700" alt="image" src="https://user-images.githubusercontent.com/48220549/147111889-1ef68346-14d9-4747-b131-10813142e9ba.png">

Reply:
The best [docs](https://pypi.org/project/sourcery-cli/) I found.

Reply:
I was specifically asking about the docs where they mention `~/.config/sourcery/auth.yaml` since I couldn't find it anywhere, because for Windows it would have to be something like `~/AppData/Local/sourcery/..`

I guess the local file would be `.sourcery.yaml`? but I can't find if it has anything to do with storing the auth token.

Reply:
I think so. I'll have to look around. But why use the path? Once it is setup(i.e i am logged in), I dont have to use the Path again. Sourcery fetches the token and saves it for future usage. I.e the LSP works everything after the initial setup.

Reply:
okay. Turns out there are no docs on this...Which is a bad thing. Opened an issue regarding this.